backend/src/search/series.rs

```diff
@@ -17,6 +18,8 @@ pub(crate) struct Series {
     pub(crate) opencast_id: String,
     pub(crate) title: String,
     pub(crate) description: Option<String>,
+    pub(crate) created: Option<DateTime<Utc>>,
+    pub(crate) created_timestamp: Option<i64>,
```

Comment:
Yeah good question. I use `created` for events and series, but `updated` for playlists. I figured `created` would be a better approximation of what users want to see first in the dropdown menu. I only used `updated` for playlists, because there is no `created` field there.

But if you'd argue that `updated` is at least as good of approximating user-expectations than `created`, I can certainly change it. Would be more consistent and we would not have to deal with `Option`